src/Rakaposhi.Core/Rakaposhi.Business.Core/DBRepository/DBRepositoryFactory.cs

```diff
@@ -24,13 +25,27 @@ public IUserStatusRepository UserStatusRepository
         {
             get
             {
-                if( _userStatusRepository is null)
+                if (_userStatusRepository is null)
                 {
                     _userStatusRepository = new UserStatusRepository();
                 }
 
                 return _userStatusRepository;
             }
         }
+
+        public ITransTypeRepository TransTypeRepository
+        {
+            get
+            {
+                if (_transTypeRepository is null)
```

Comment:
Remove space between if and its brackets.

Reply:
fixed